src/Tizen.NUI.Components/Attributes/ViewAttributes.cs

```diff
@@ -264,5 +280,89 @@ public override Attributes Clone()
             return new ViewAttributes(this);
         }
 
+        /// <summary>
+        /// Apply ViewAttributes to View.
+        /// </summary>
+        public virtual void ApplyToView(View view, ControlStates state = ControlStates.Normal)
+        {
+            if (view == null)
+            {
+                return;
+            }
+
+            if (Position != null)
+            {
+                view.Position = Position;
+            }
+            if (Size != null)
+            {
+                view.Size = Size;
+            }
+            if (MinimumSize != null)
+            {
+                view.MinimumSize = MinimumSize;
+            }
+            if (PositionUsesPivotPoint != null)
+            {
+                view.PositionUsesPivotPoint = PositionUsesPivotPoint.Value;
+            }
+            if (ParentOrigin != null)
+            {
+                view.ParentOrigin = ParentOrigin;
+            }
+            if (PivotPoint != null)
+            {
+                view.PivotPoint = PivotPoint;
+            }
+            if (WidthResizePolicy != null)
+            {
+                view.WidthResizePolicy = WidthResizePolicy.Value;
+            }
+            if (HeightResizePolicy != null)
+            {
+                view.HeightResizePolicy = HeightResizePolicy.Value;
+            }
+            if (SizeModeFactor != null)
+            {
+                view.SizeModeFactor = SizeModeFactor;
+            }
+
+            if (!appliedViews.ContainsKey(view))
+            {
+                appliedViews.Add(view, 0);
+                RegisterStateChangeCallback(view);
+            }
+        }
+
+        protected virtual void RegisterStateChangeCallback(View view)
+        {
+            view.FocusGained += (object sender, global::System.EventArgs e) =>
+            {
+                ApplyToView(view, ControlStates.Focused);
+            };
+
+            view.FocusLost += (object sender, global::System.EventArgs e) =>
+            {
+                ApplyToView(view, ControlStates.Normal);
+            };
+        }
+
+        public void DisApplyToView(View view)
+        {
+            if (appliedViews.ContainsKey(view))
+            {
+                appliedViews.Remove(view);
```

Comment:
I wonder if there's no need to unregister FocusGained and FocusLost callback from the view here.

Reply:
If don't unregister the callback, when some Views are disposed, the changing of theme will try to apply the attribute to disposed View, it will cause issue.

Reply:
I don't understand. I was talking about unregistering `FocusGained` and `FocusLost` callback from the `view`. But what you said is about he reason of removing `view` from the `appliedViews`.

Reply:
I was talking about unregistering `FocusGained` and `FocusLost` callbacks from the `view`. 
I think you should unregister  `FocusGained` and `FocusLost` callbacks from the `view` here.

But what you commented is the reason of removing `view` from the `appliedViews`.

Reply:
I'm sorry I misunderstood you， the FocusGained and FocusLoast will be removed when Disapply.